spec/directions/bake_chapter_key_concepts_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeChapterKeyConcepts do
+  before do
+    stub_locales({
+      'eoc_key_concepts': 'Key Concepts'
+    })
+  end
+
+  let(:chapter) do
+    chapter_element(
+      page_element(
+        <<~HTML
+          <div data-type="document-title">Page 1</div>
+          <section id="auto_7c_fs-id1" class="key-concepts">
+            <h3 data-type="document-title">WWF History</h3>
+            <p>Concepts blah.</p>
+          </section>
+        HTML
+      )
+    )
+  end
+
+  it 'works' do
+    expect(
+      described_class.v1(chapter: chapter, metadata_source: metadata_element)
+    ).to match_normalized_html(
+      <<~HTML
+        <div data-type="chapter">
+          <div data-type="page">
+            <div data-type="document-title">Page 1</div>
+            <section id="auto_7c_fs-id1" class="key-concepts">
+              <a href="#auto_7c_0">
+                <h3 data-type="document-title" id="auto_7c_0_copy_XXX">
+                  <span class="os-number">1.2</span>
+                  <span class="os-divider"> </span>
+                  <span class="os-text" data-type="" itemprop="">Page 1</span>
+                </h3>
+              </a>
+              <p>Concepts blah.</p>
```

Comment:
Can you check that in the baked output this stuff is duplicated here and below?

Reply:
yeah it looks like this because it does `chapter append` not `replace`